### PR TITLE
Add line break for warning for policy/asset/liability (Fix Issue #317)

### LINKED ISSUE #317: Warning for duplicate policy / asset /liability should have a line break.
image

Shouldn't the warning be on a new line for readability + consistency with how warnings are generally done in SE (Original, non-warning output, then in a new line your warning notice).

### PR DESCRIPTION
**Pull request by DonnyS57**
Fixes #317

### CODE REVIEW

**Review by sheehui — DISMISSED** [review]
nice!

File: src/main/java/donnafin/logic/commands/AppendCommand.java [hunk 1]
```diff
@@ -54,11 +54,11 @@
     public static final String MESSAGE_DUPLICATE_POLICY = "Sorry, the policy with this exact specifications"
```
Comment (sheehui) [inline src/main/java/donnafin/logic/commands/AppendCommand.java:54]: it should be 'a policy with...'. can do the same for assets and liability?

Comment (DonnyS57) [reply src/main/java/donnafin/logic/commands/AppendCommand.java:54]: oki

**Review by bharathcs — APPROVED** [review]
LGTM!

**Pull request merged**